### PR TITLE
Fix/update rx bloc generator dependencies

### PR DESCRIPTION
**Pull request by DDavidPrime**
Addresses #276

### CODE REVIEW

File: packages/rx_bloc_generator/CHANGELOG.md [hunk 1]
```diff
@@ -1,3 +1,7 @@
+## [5.2.0] - July 27, 2022
+* Update dependencies
+* [BREAKING CHANGE] Update analyzer to ">=3.0.0 <=5.0.0"
```
Comment (StanevPrime) [inline packages/rx_bloc_generator/CHANGELOG.md:3]: According to the semantic versioning if there is a breaking change the major version should be increased. If that's the case, please bump the package version to 6.0.0

File: packages/rx_bloc_generator/lib/src/builders/event_arguments_class.dart [hunk 2]
```diff
@@ -17,7 +17,7 @@ class _EventArgumentsClass implements _BuilderContract {
         (b) => b
           ..docs.addAll([
             '/// Helps providing the arguments in the [Subject.add] for',
-            '/// [${method.enclosingElement.name ?? ''}.${method.name}] event'
+            '/// [${method.enclosingElement2.name ?? ''}.${method.name}] event'
```
Comment (StanevPrime) [inline packages/rx_bloc_generator/lib/src/builders/event_arguments_class.dart:20]: What's the reason of changing **enclosingElement** to **enclosingElement2** ?

File: packages/rx_bloc_generator/pubspec.yaml [hunk 3]
```diff
@@ -1,6 +1,6 @@
 name: rx_bloc_generator
 description: Code generator for rx_bloc that makes your BloC zero-boilerplate.
-version: 5.1.1
+version: 5.2.0
```
Comment (StanevPrime) [inline packages/rx_bloc_generator/pubspec.yaml:3]: Please check the comment above

File: packages/rx_bloc_generator/lib/src/builders/event_arguments_class.dart (hunk 2 above)
Comment (DDavidPrime) [reply packages/rx_bloc_generator/lib/src/builders/event_arguments_class.dart:20]: Honestly, beats me why they chose to name the newer field like that. The suggested replacement for `enclosingElement` (which was deprecated) was `enclosingElement2`.

File: packages/rx_bloc_generator/CHANGELOG.md (hunk 1 above)
Comment (DDavidPrime) [reply packages/rx_bloc_generator/CHANGELOG.md:3]: I wasn't aware of such convention. Will change it to `6..0.0`.

Comment (StanevPrime) [reply packages/rx_bloc_generator/CHANGELOG.md:3]: Please add a bullet point describing this new feature - https://github.com/Prime-Holding/rx_bloc/pull/197

**Review by StanevPrime — CHANGES_REQUESTED** [review]
Please add a bullet point describing this new feature - https://github.com/Prime-Holding/rx_bloc/pull/197

**Review by StanevPrime — APPROVED** [review]

**Pull request merged**